
std_msgs/Float32MultiArray wout_x
std_msgs/Float32MultiArray wout_y

float32 wout_mean
float32 wout_min
float32 wout_max
